sourceaddrs/source_registry.go

```diff
@@ -29,6 +29,12 @@ type RegistrySource struct {
 	subPath string
 }
 
+type RegistryVersionDeprecation struct {
```

Comment:
This type may be in the wrong package, too.

Reply:
I like the way you un-nested the thing previously called "RemoteSourceInfo"!

Reply:
Ah gotcha. Just to clarify your thinking, would you consider it out of place since it's only used in `sourcebundle`?

Reply:
And in any case any suggestions where it might be a better fit? No file in particular pops out at me in the package.

Reply:
Ended up moving it to `package_meta.go` in the sourcebundle package, seems to be the best fit.